transforms/universal/fdedup/ray/pyproject.toml

```diff
@@ -6,10 +6,11 @@ description = "fdedup Ray Transform"
 license = {text = "Apache-2.0"}
 readme = {file = "README.md", content-type = "text/markdown"}
 authors = [
-    { name = "David Wood", email = "dawood@us.ibm.com" },
-    { name = "Boris Lublinsky", email = "blublinsky@ibm.com" },
+    { name = "Nelson Bore", email = "k.nelsonbore@gmail.com" },
+    { name = "Constantin Adam", email = "cmadam@us.ibm.com" },
 ]
 dependencies = [
+    "dpk_fdedup_transform_python==0.2.2.dev1",
     "data-prep-toolkit-ray==0.2.2.dev1",
```

Comment:
Should be using data-prep-toolkit[ray] and also once the branch is updated with dev, this will change to 0.2.2.dev2

Reply:
Fixed in commit https://github.com/IBM/data-prep-kit/pull/699/commits/2f61be7938d7540a0a1831e85b8a961bef24d35c